frontend/apps/marketing/src/components/iframe/iframeContentfulDefinition.ts

```diff
@@ -0,0 +1,48 @@
+// Creates a definition for the Iframe component to be used in Contentful Studio
+import {ComponentDefinition} from '@contentful/experiences-sdk-react';
+
+export const IframeContentfulComponentDefinition: ComponentDefinition = {
+  id: 'iframe',
+  name: 'iFrame',
+  category: 'Advanced',
+  thumbnailUrl:
+    'https://images.ctfassets.net/90t6bu6vlf76/1qy9FC9Bqb4ADrpyszIa5M/5e791afb110fc5bc8caa95710c1dea16/component_code_thumbnail.png',
+  tooltip: {
+    description:
+      'Embed external content using an iframe. Ideal for embedding videos, interactive tools, or third-party widgets within a page.',
```

Comment:
@moshebaricdo since this will primarily be used for Pardot forms I added forms and removed videos since we have the Video component (and we don't want this to be used for videos). 
```suggestion
      'Embed external content using an iframe. Ideal for embedding forms, interactive tools, or third-party widgets within a page.',
```